### PR TITLE
Add Radiant Historia: Perfect Chronology

### PR DESCRIPTION
**Pull request by Xemiru**
Adds Radiant Historia: Perfect Chronology.

*Automatically generated by PR builder*

### CODE REVIEW

**Comment by Xemiru** [comment]
There're `EUR` and `JPN` releases of the game, though I can't find their title IDs right now.

**Comment by robzombie91** [comment]
00040000001CD900 - eur
00040000001A0C00 - jpn

File: games/radiant-historia-perfect-chronology/game.dat [hunk 1]
```diff
@@ -0,0 +1,9 @@
+title = "Radiant Historia: Perfect Chronology"
+description = "Radiant Historia: Perfect Chronology is a role-playing game released by Atlus. It is an expanded remake of the NDS game of the same title originally released in February of 2011.\n\nThe continent of Vainqueur is plagued with the spreading desertification, laying the land to waste by turning it slowly into sand. On a mission, Stocke discovers the powers of the White Chronicle, and will need to right the wrongs of the past in order to prevent the total annihilation of the world."
```
Comment (Leo626) [inline games/radiant-historia-perfect-chronology/game.dat]: The description is too long and should be objective. You can use the one from [here](http://radianthistoria.wikia.com/wiki/Radiant_Historia:_Perfect_Chronology) but without the Japanese name and line breaks (\n).

Comment (robzombie91) [inline games/radiant-historia-perfect-chronology/game.dat]: description = "Radiant Historia: Perfect Chronology is a 3ds enhanced port role-playing game developed and published by Atlas.

**Comment by robzombie91** [comment]
http://www.gamereleasedates.net/images/covers/3ds/cover-3ds-radiant-historia-perfect-chronology.jpg

**Comment by Xemiru** [comment]
I feel I should remove that part about Nemesia and the Subhistory since it feels like a partial content spoiler. I know I didn't appreciate seeing it, at least; I didn't know she was involved.

**Comment by Leo626** [comment]
I'm fine with that. Still need to create the wiki entry.

**Comment by Xemiru** [comment]
The wiki entry is available [here](https://github.com/citra-emu/citra-games-wiki/wiki/Radiant-Historia-Perfect-Chronology).

**Pull request merged**